Libraries/Components/StatusBar/StatusBar.js

```diff
@@ -0,0 +1,159 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule StatusBar
+ * @flow
+ */
+'use strict';
+
+const React = require('React');
+const ColorPropType = require('ColorPropType');
+const Platform = require('Platform');
+
+const processColor = require('processColor');
+
+if (Platform.OS === 'ios') {
+  var RCTStatusBarManager = require('NativeModules').StatusBarManager;
+} else if (Platform.OS === 'android') {
+  var RCTStatusBarManager = require('NativeModules').StatusBarAndroid;
+}
+
+import type ReactElement from 'ReactElement';
+
+type DefaultProps = {
+  animated: boolean;
+};
+
+const StatusBar = React.createClass({
+  statics: {
+    _propsStack: [],
+  },
+
+  propTypes: {
+    /**
+     * If the status bar is hidden.
+     */
+    hidden: React.PropTypes.bool,
+    /**
+     * If the transition between status bar property changes should be animated.
+     * Supported for color, barStyle and hidden.
+     */
+    animated: React.PropTypes.bool,
+    /**
+     * The color of the status bar.
+     * @platform android
+     */
+    color: ColorPropType,
+    /**
+     * If the status bar is translucent.
+     * When translucent is set to true, the app will draw under the status bar.
+     * This is useful when using a semi transparent status bar color.
+     *
+     * @platform android
+     */
+    translucent: React.PropTypes.bool,
```

Comment:
Can this property be inferred from the backgroundColor? If it has an alpha component < 1.0 then it's translucent, otherwise it's inherently opaque.